### PR TITLE
Fix asset minifying.

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jaycee** [comment]
QA:

Run `make clean-all fast-babel qa-server` and ensure the gui comes up to the canvas w/o errors.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3762/
Test PASSed.

File: qa.ini [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+[app:main]
+use = egg:jujugui
+
+pyramid.reload_templates = true
+pyramid.default_locale_name = en
+
+jujugui.sandbox = true
+jujugui.raw = false
+jujugui.combine = false
+
```
Comment (bac) [inline qa.ini]: combine should be True

File: Makefile [hunk 2]
```diff
@@ -141,7 +146,7 @@ $(BUILT_JS_ASSETS): $(NODE_MODULES)
 	cp -Lr $(JS_ASSETS) $(GUIBUILD)/app/assets/
 	find $(BUILT_JS_ASSETS) -type f -name "*.js" \
 		-not -name "react*" \
-		-not -name "*d3-wrapper*" \
+		-not -name "*d3-wrapper*" | \
```
Comment (bac) [inline Makefile]: Amazed you found that.  Good job.

Too bad find doesn't return an error code.

**Comment by bac** [comment]
:+1: , no QA

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3763/
Test PASSed.

**Comment by bac** [comment]
Doing QA now

**Comment by bac** [comment]
QA-OK (now land it)

**Comment by jaycee** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3765/
Test PASSed.

**Pull request merged**